package.json

```diff
@@ -71,7 +72,7 @@
     "wdio-mocha-framework": "0.3.1",
     "webdriverio": "4.0.9",
     "whatwg-fetch": "0.10.1",
-    "yuidoc-lucid-theme": "git+ssh://git@github.com:Shopify/yuidoc-lucid-theme.git#c9c62ba",
+    "yuidoc-lucid-theme": "git+ssh://git@github.com/Shopify/yuidoc-lucid-theme.git#c9c62ba",
```

Comment:
`yarn install` failed with the colon (https://github.com/yarnpkg/yarn/issues/513#issuecomment-254850558) :(